clintrials/dosefinding/efftox.py

```
@@ -344,48 +344,78 @@ def __call__(self, prob_eff, prob_tox):
         """
         x = prob_eff
         y = prob_tox
-        if np.all(0 < x) and np.all(x < 1) and np.all(0 < y) and np.all(y < 1):
+        if np.all(0 <= x) and np.all(x <= 1) and np.all(0 <= y) and np.all(y <= 1):
```

Ruff (0.15.18)

[warning] Yoda condition detected

Rewrite as `x >= 0`

(SIM300)